core/src/main/java/org/apache/druid/math/expr/Function.java

```diff
@@ -76,6 +76,22 @@
     return Collections.emptySet();
   }
 
+  /**
+   * Returns true if a function takes an array arguments
```

Comment:
1. "takes array arguments" or "takes an array argument" (grammar, and also, I'm not sure which one you mean; the two phrases mean different things)
2. Clarify intent: what does "takes an array arguments" mean? That it _might_ take array arguments? Or that it _must_ take array arguments?

Reply:
the typo is that it's supposed to be `any array arguments` heh, which I think clears up both things if I fix?